pkg/distributor/ha_tracker.go

```diff
@@ -88,7 +88,12 @@ func (cfg *HATrackerConfig) Validate() error {
 		return fmt.Errorf(errInvalidFailoverTimeout, cfg.FailoverTimeout, minFailureTimeout)
 	}
 
-	return nil
+	// Tracker kv store only supports consul and etcd.
+	if cfg.KVStore.Store == "consul" || cfg.KVStore.Store == "etcd" {
```

Comment:
Would you mind to add a "allowed list" of supported stores so it would be easier to add more than in the future without having to do extra checks?